src/SurveyService.Models.Dto/Requests/Option/CreateOptionRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.SurveyService.Models.Dto.Requests.Option;
+
+public record CreateOptionRequest
+{
+  public Guid QuestionId { get; set; }
+  [Required]
```

Comment:
It would be great to write error message here.

Reply:
mmm.. Project's code style is to add no messages to "Required", isn't it?

Reply:
> mmm.. Project's code style is to add no messages to "Required", isn't it?
I thought, that we have to add some desription for ValidationAttributes, otherwise someone will get only error without a description. #
https://docs.microsoft.com/ru-ru/dotnet/api/system.componentmodel.dataannotations.requiredattribute?view=net-6.0
It's the same as adding .WithMessage in Validator. Isn't it?

Reply:
Ok